code/modules/antagonists/heretic/heretic_knowledge.dm

```diff
@@ -710,3 +710,19 @@
 		sacrifice.gib()
 
 	return ..()
+
+/datum/heretic_knowledge/mansus_restoration_ritual
+	name = "Mansus restoration" // sounds awful
+	desc = "Sacrifice lungs, potted plant and a radio to restore your link to the Mansus."
+	required_atoms = (
+		/obj/item/organ/inter/lungs,
+		/obj/item/kirbyplants, // because why not lol
+		/obj/item/radio,
+	)
+	priority = MAX_KNOWLEDGE_PRIORITY - 4 // below basic rituals so you can see it
+	route = null
+
+/datum/heretic_knowledge/mansus_restoration_ritual/on_finished_recipe
+	var/datum/antagonist/heretic/our_heretic = IS_HERETIC(user)
+	ADD_TRAIT(our_heretic, TRAIT_ALLOW_HERETIC_CASTING, src)
```

Comment:
Here you are just giving a heretic a permanent focus, without a need of wearing a focus item.